src/chrome/content/rules/DW.com.xml

```diff
@@ -0,0 +1,45 @@
+<!--
+	Invalid certificate:
+		- multimedia.dw.com
+		- nl.dw.com
+		- webmo.dw.com
```

Comment:
`webmo.dw.com` no longer serve an invalid certificate, would you add it to the list of `target` and update the comments?

Reply:
No   NOT TRUSTED
https://www.ssllabs.com/ssltest/analyze.html?d=webmo.dw.com&latest